keyboards/ryanbaekr/rb86/readme.md

```diff
@@ -0,0 +1,13 @@
+# rb86
+
+A diy sandwich compact TKL keyboard. 
+
+* Keyboard Maintainer: [ryanbaekr](https://github.com/ryanbaekr)
+* Hardware Supported: rb86 with Elite-C
+* Hardware Availability: 2021/05/15
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to jump into the bootloader, to flash new firmware?